src/server/index.jsx

```diff
@@ -166,12 +166,13 @@ server
         } catch ({ message }) {
           logger.error(`Error fetching Cosmos dials: ${message}`);
         }
+        // Preserve initial dial state in window so it is available during hydration
+        data.dials = dials;
```

Comment:
I'll need to look into alternatives to this approach of adding dials to the data object. I'd rather it was a separate to data, but also ensure it persists and is available during hydration.

Reply:
OK - my thinking is here: https://github.com/bbc/simorgh/pull/2283#discussion_r300298597

Reply:
Also: I noticed we already have a `status` property under `data` which is separate to the actual page data too.

Reply:
Thanks. Sounds good!